api/migrations/0009_item_model_number.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.0.1 on 2018-03-05 09:38
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0008_auto_20180301_1625'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='item',
+            name='model_number',
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
```

Comment:
Looks like you have not changed the migration file to be `models.PROTECT`

Reply:
i think that's catered for [here](https://github.com/AndelaOSP/art-backend/pull/30/files#diff-e6a82ebaf34ad7d12411085a5838ea38R17)

Reply:
I see. What I have been doing is deleting the migration file, but this is fine.